graylog2-web-interface/src/components/indices/IndexMaintenanceStrategiesConfiguration.jsx

```diff
@@ -152,36 +152,55 @@ const IndexMaintenanceStrategiesConfiguration = ({
   const retentionIsNotNoop = retentionStrategyClass !== NOOP_RETENTION_STRATEGY;
   const shouldShowMaxRetentionWarning = maxRetentionPeriod && rotationStrategyClass === TIME_BASED_ROTATION_STRATEGY && retentionIsNotNoop;
 
+  function getDescription() {
+    if (description) {
+      return (
+        <StyledAlert>
+          <Icon name="info-circle" />{' '} {description}
+        </StyledAlert>
+      );
+    }
+
+    return null;
+  }
+
   return (
     <span>
       <StyledH3>{title}</StyledH3>
-      <StyledAlert>
-        <Icon name="info-circle" />{' '} {description}
-      </StyledAlert>
+      {getDescription()}
       {shouldShowMaxRetentionWarning && (
-      <StyledAlert bsStyle="warning">
-        <Icon name="exclamation-triangle" />{' '} The effective retention period value calculated from the <b>Rotation period</b> and the
-        <b> max number of indices</b> should not be greater than the <b>Max retention period</b> of <b>{maxRetentionPeriod}</b> set by the Administrator.
-      </StyledAlert>
+        <StyledAlert bsStyle="warning">
+          <Icon name="exclamation-triangle" />{' '} The effective retention period value calculated from the
+          <b>Rotation period</b> and the <b>max number of indices</b> should not be greater than the
+          <b>Max retention period </b> of <b>{maxRetentionPeriod}</b> set by the Administrator.
+        </StyledAlert>
       )}
-      <Input id="strategy-select"
-             labelClassName="col-sm-3"
-             wrapperClassName="col-sm-9"
-             label={selectPlaceholder}>
-        <StyledSelect placeholder={selectPlaceholder}
-                      options={_availableSelectOptions()}
-                      matchProp="label"
-                      value={_activeSelection()}
-                      onChange={_onSelect} />
-      </Input>
-      {_getConfigurationComponent(_activeSelection(), pluginExports, strategies, strategy, config, _onConfigUpdate)}
+      <Row>
+        <Col md={12}>
+          <Input id="strategy-select"
+                 labelClassName="col-sm-3"
+                 wrapperClassName="col-sm-9"
+                 label={selectPlaceholder}>
+            <StyledSelect placeholder={selectPlaceholder}
+                          options={_availableSelectOptions()}
+                          matchProp="label"
+                          value={_activeSelection()}
+                          onChange={_onSelect} />
```

Comment:
```suggestion
                          onChange={_onSelect}
                          clearable={false} />
```
The Select components can currently be cleared, but the rotation and retention configurations are required. You can save the form with the two selects cleared which does nothing to the existing config. Suggest making them not clearable to avoid this happening.